hack/scripts/run-non-olm-kuttl-test.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
+
+export NON_OLM="true"
+
+ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+sequential_suite=$ROOT/../../test/openshift/e2e/sequential/
+parallel_suite=$ROOT/../../test/openshift/e2e/parallel/
+
+testsuite="sequential"
+
+# Get test suite argument
+while getopts ":t:" opt; do
+  case ${opt} in
+    t) testsuite=$OPTARG;;
+    \?) echo "Please provide options sequential or parallel -$OPTARG" >&2;;
+  esac
+done
+
+# these tests will be removed while running non-olm operator test
+# 1-031_validate_toolchain
+# 1-085_validate_dynamic_plugin_installation
+# 1-036_validate_keycloak_resource_reqs
+# 1-038_validate_productized_images
+# 1-051-validate_csv_permissions
+# 1-073_validate_rhsso
+# 1-077_validate_disable_dex_removed
+# 1-090_validate_permissions
+
+if [ -n "$EXCLUDED_TESTS" ]; then
+  filenames=$EXCLUDED_TESTS
+else 
+  filenames="1-031_validate_toolchain 1-085_validate_dynamic_plugin_installation 1-036_validate_keycloak_resource_reqs 1-038_validate_productized_images 1-051-validate_csv_permissions 1-073_validate_rhsso 1-077_validate_disable_dex_removed 1-090_validate_permissions"
```

Comment:
Do we need to exclude this tests irrespective of what tests user excludes as they would not work on non-OLM?

Reply:
If the user specifies the tests to be removed in EXCLUDED_TESTS then we remove only whatever they specify (if he doesn't mention a failing test, then that test will be run and will fail), else we exclude all hard coded ones. And yes, they don't work with NON_OLM.

Reply:
If the tests would not execute successfully in NON_OLM, should we just ignore them by default? I do not see a reason to even execute this tests if they are sure to fail on NON_OLM, especially as this script is to execute tests on non-olm suite.

Reply:
Yes,but there is no ignore option in kuttl for the test directories.

Reply:
We can do something like below by ignoring / removing NON_OLM tests always.

``` suggestion
filenames="1-031_validate_toolchain 1-085_validate_dynamic_plugin_installation 1-036_validate_keycloak_resource_reqs 1-038_validate_productized_images 1-051-validate_csv_permissions 1-073_validate_rhsso 1-077_validate_disable_dex_removed 1-090_validate_permissions"

if [ -n "$EXCLUDED_TESTS" ]; then
  filenames="${filenames} ${EXCLUDED_TESTS}"
fi
```

Let me know what you think?

Reply:
Thanks, yes this can be done

Reply:
@saumeya I think we missed updating this entire if-else block with the suggestion.